server/controllers/allocationHistoryController.ts

```diff
@@ -2,13 +2,70 @@ import { Request, Response } from 'express'
 import WorkloadService from '../services/workloadService'
 import ProbationEstateService from '../services/probationEstateService'
 import config from '../config'
+import UserPreferenceService from '../services/userPreferenceService'
+import AllocationsService from '../services/allocationsService'
 
 export default class AllocationHistoryController {
   constructor(
     private readonly workloadService: WorkloadService,
-    private readonly probationEstateService: ProbationEstateService
+    private readonly probationEstateService: ProbationEstateService,
+    private readonly userPreferenceService: UserPreferenceService,
+    private readonly allocationService: AllocationsService
   ) {}
 
+  async getCasesAllocatedByTeam(req: Request, res: Response, pduCode): Promise<void> {
+    const { token, username } = res.locals.user
+    const [pduDetails, teamsUserPreference] = await Promise.all([
+      this.probationEstateService.getProbationDeliveryUnitDetails(token, pduCode),
+      this.userPreferenceService.getTeamsUserPreference(token, username),
+    ])
+    const teamCodes = teamsUserPreference.items
+    const teams = await this.probationEstateService.getTeamsByCode(token, teamCodes)
+    const caseAllocationHistory = await this.workloadService.postAllocationHistory(
+      token,
+      config.casesAllocatedSinceDate().toISOString(),
+      teamCodes
+    )
+    const caseAllocationDetails = await Promise.all(
+      caseAllocationHistory.cases.map(async caseAllocation => {
+        const laoStatus = await this.allocationService.getLAOStatusforAllocation(token, caseAllocation.crn)
+        return {
+          name: caseAllocation.name.combinedName,
+          crn: caseAllocation.crn,
+          tier: caseAllocation.tier,
+          allocationDate: caseAllocation.allocatedOn,
+          staffName: caseAllocation.staff?.name?.combinedName,
+          staffCode: caseAllocation.staff?.code,
+          pduCode,
+          allocatedBy: caseAllocation.allocatingSpo,
+          team: caseAllocation.teamCode,
+          apopExcluded: laoStatus.isRestricted,
+          excluded: laoStatus.isRedacted,
```

Comment:
These are the wrong way round. Should be
 **apopExcluded: laoStatus.isRedacted
 excluded: laoStatus.isRestricted**